docs/cloud.md

```diff
@@ -0,0 +1,234 @@
+
+
+```
+# File name: eks-config.yaml
+
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: feldera-eks
+  region: us-west-1
+  version: "1.27"
+
+# OIDC provider is needed to use EBS
+iam:
+  withOIDC: true
+
+managedNodeGroups:
+  - name: managed-ng
+    desiredCapacity: 3
+    instanceType: m5.large
+    privateNetworking: true
+
+vpc:
+  clusterEndpoints:
+    privateAccess: true
+    publicAccess:  true
+    # publicAccessCIDRs: ["<a.b.c.d/32>"] # Change to your IP address if using publicAccess: true
+
+addons:
+- name: aws-ebs-csi-driver
+  wellKnownPolicies:
+    ebsCSIController: true
+```
+
+To create an EKS cluster with this configuration, run:
+
+```
+eksctl create cluster -f eks-config.yaml
+```
+
+This uses CloudFormation behind-the-scenes to bring up an EKS cluster named
+`feldera-eks` with three worker nodes (m5.large EC2 instances, specified in
+managedNodeGroups), running in a newly created dedicated VPC. It should take
+roughly 15-20 minutes.
+
+`eksctl` will create the VPC in the `us-west-1` region which will run the EKS
+cluster. The cluster will be deployed across two availability zones (AZs). Each
+AZ will have two subnets, one private and one public. The private subnets will
+have outbound Internet connectivity via a NAT gateway. The public subnets will
+have inbound and outbound connectivity via an Internet Gateway. The worker
+nodes will only be on the private subnet.
+
+
+:::tip
+Unless you are running `eksctl` from a machine which is in the same VPC and
+subnet as the EKS control plane, you will want to set `publicAccess: true`.
+If so, we strongly recommend using the `publicAccessCIDRs` setting to limit IP ranges
+which are allowed to interact with the EKS control plane. For example, set it
+to the publicly visible IP of the machine where you are running these commands.
+
+See the [eksctl cluster access](https://eksctl.io/usage/vpc-cluster-access/)
+documentation to learn more.
+:::
+
+:::tip
+`eksctl` allows extensive configuration beyond what we've shown here. Check out
+the [configuration file schema](https://eksctl.io/usage/schema/) to learn more.
+:::
+
+To verify that the EKS cluster is running, run:
+
+```
+kubectl get nodes
+```
+
+You should see a three node cluster, all with the status `Ready`.
+
+## Cluster ingress
+
+To allow external access to services running inside the EKS cluster, we
+recommend setting up a Kubernetes Ingress controller. This is useful to allow
+access to the Feldera Web Console or to issue REST API calls directly. 
+
+:::tip
+When you install an ingress controller, it uses an AWS load balancer to expose
+an IP address via a generated DNS name for external access. By default,
+creating ingress controllers leads to both an internal and external load
+balancer being created, with the external one being prefered to expose
+services. Most Feldera Cloud users should prefer to only expose  services only
```

Comment:
> Most Feldera Cloud users should prefer to only expose  services only an internal load balancer.

Most Feldera Cloud users should need to expose Feldera services only to an internal load balancer.